packages/artifact/__tests__/upload.test.ts

```diff
@@ -174,6 +179,43 @@ describe('Upload Tests', () => {
     expect(uploadResult.uploadSize).toEqual(expectedTotalSize)
   })
 
+  const onLinuxIt = platform() === 'linux' ? it : it.skip
```

Comment:
Is there are a reason why this test is only on Linux and are there any platform limitations that can arise? We try to remain platform agnostic as much as possible.

I think it should also be able to at least run on Mac just fine (on Windows named pipes are a considerably different with the `\\.\pipe\` file location so I think it's fine to avoid a test for that) but I think you could do something like:

`const notWindowsIt = process.platform !== 'win32'`

Reply:
Thanks, good point. I think I can do one better, there is also other Unixes where this should work, so testing if `mkfifo` exists is better. Now there is a test with `which mkfifo` to see if `mkfifo` can be found.